app/controllers/albums.js

```diff
@@ -0,0 +1,21 @@
+const { getAllAlbums, getAlbumById, getPhotosByAlbum } = require('../services/externalServices');
+
+const sendResponse = res => response => {
+  res.status(response.status || 200);
+  res.json(response);
```

Comment:
I don't see this abstraction like too necessary, could be more simple resolve this in controller directly like `res.send(response)` (default status code is 200 and response is sent as JSON)